### DISCUSSION TITLE
about v.1.0.0

### DISCUSSION THREAD

**Discussion by kiya52** [chidiwilliams/buzz#840] [Q&A]
Dear chidiwilliams,

Thanks for yout update v.1.0.0 of buzz in the operation system of windows.
I try to transcribe a chinese voice file to text. 
It is good to use this software by models of whisper tiny, small, medium. and the transcribe speed is the same as before.
But it can't be work any models of fast whisper. (I have already downloaded the fast whisper model)

it will show (failed (unknown error))

B.R.

Kiya, Chang

**Comment by raivisdejus** [comment]
To identify the problem with Faster Whisper paste this in address bar of the Windows `%USERPROFILE%\AppData\Local\Buzz\Buzz\Logs` There will be a `logs` file with information about what happened on the system when it failed. Information in this file can help us identify the problem, please send it here.

Also you can try Whisper.cpp it works faster than regular Whisper
